tools/Render1D.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python
```

Comment:
This looks good. Last couple remaining things:
- Can you move this to `src/Visualization/Python` and add to the CMake file there:
  ```cmake
  spectre_python_add_executable(
    Render1D
    Visualization/Render1D.py
    )
  ```
  as well as add it to the list of `PYTHON_FILES`?
- Once you've moved it, can you make this file executable by running `chmod +x Render1D.py`?
- Your commit message is currently:
  ```
  Added 1D data visualisation using python

  fixup
  ```
  Can you remove the "fixup"? This can be down with an interactive rebase where instead of doing "s" or "f" you do "r" for "reword". We can do this in person on Friday too :)

Reply:
You're still adding this file into `tools` in addition to `src/Visualization/python`